model/bin/w3_setup

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -e
+#!/bin/bash -eu
```

Comment:
Is this file still used/needed with the transition to CMAKE?

Reply:
I guess I'm just curious that it's not either left unchanged (to support both builds) or deleted.

Reply:
No, changes to this file can be ignored. Not sure why I added this. I was having issues getting this to work on my Mac, so I changed getopts and this.

Reply:
Instead of asking reviewers to ignore changes, replace these files on your branch with copies from master, then they will disappear from code review.

Reply:
If these changes are needed for running on a Mac (and this file won't be going away), we could add them to the main repo in another PR.